<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>427</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="MplWidget" name="MplWidget" native="true">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>0</y>
           <width>491</width>
           <height>311</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="QPushButton" name="move_pushButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>330</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Move</string>
         </property>
        </widget>
        <widget class="QPushButton" name="engage_pushButton">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>330</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Engage</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="console">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>330</y>
           <width>351</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QTreeWidget" name="treeFilter">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>301</height>
          </rect>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>My Company</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <item>
           <property name="text">
            <string>Soldiers</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>APC's</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Other Companies</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Enemies</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <item>
           <property name="text">
            <string>Soldiers</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Launchers</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lookout Points</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
         </item>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
